lib/compilers/java.js

```diff
@@ -246,6 +246,14 @@ export class JavaCompiler extends BaseCompiler {
                         text: codeLineCandidate.trimEnd(),
                     });
                 } else {
+                    const isClosingCurlyBrace = codeLineCandidate.match(/\s+}/);
```

Comment:
Maybe we should specify that we're attempting to match the closing bracket of a lookupswitch here?
```suggestion
                } else {
                    // Attempt to match the closing } of a lookupswitch. If we don't include the closing bracket, then
                    // the brackets will be misaligned, and it may be confusing to read.
                    const isClosingCurlyBrace = codeLineCandidate.match(/\s+}/);
```